paasta_tools/setup_tron_namespace.py

```diff
@@ -62,22 +71,48 @@ def main():
     else:
         logging.basicConfig(level=logging.WARNING)
 
-    try:
-        new_config = tron_tools.create_complete_config(
-            service=args.service,
-            soa_dir=args.soa_dir,
-        )
+        if args.all_namepsaces:
+            try:
+                services = tron_tools.get_tron_namespaces_for_cluster()
```

Comment:
do we need mutual exclusion between args.services and args.all_namespaces? or a log line to tell the user whats happening?